src/entities/User.ts

```diff
@@ -21,6 +22,7 @@ export interface APIUser {
 	accountStatus: AccountStatus;
 	accountType: AccountType;
 	profile?: APIProfile;
+	report?: APIReport;
```

Comment:
Can you remove report from APIUser? It shouldn't be sent around the API and is more private info.